dataCollection/sensors/ColorMatcher.java

```diff
@@ -63,6 +65,10 @@ private Color getTargetColor() {
 
   public ColorMatcher() {
     init();
+    COLORMATCHER_RED = "ColorMatcher/Red";
```

Comment:
maybe as a separate task but we should have a port number or some identifier associated with these in case we have multiple color sensors

Reply:
bump

Reply:
That makes sense, although we should probably make that change on the 2024-updated branch

Reply:
add a trello card